### PR TITLE
Fix: Bad alignment of button icons when using the original baseset.

### LINKED ISSUE #10197: [Bug]: Road vehicles icon displaced on original_windows baseset
### Version of OpenTTD

OpenTTD 13.0-beta2 Windows10

### Expected result

Correct icon layout as in previous versions
изображение

### Actual result

The road transport icon moved up in the menu bar
изображение

### Steps to reproduce

1. Install original_windows baseset
2. Start the game

### PR DESCRIPTION
**Pull request by michicc**
## Motivation / Problem

After the recent GUI scaling features, some button icons of the original TTD baseset have moved positions.
This looks visually less appealing than the previous look.

## Description

`orig_extra.grf`  is extended with a copy of the icons that were padded to restore the previous alignment.
There is ample precedent with this special GRF file of including TTD graphics, so one more should not be of any more or less concern.

This fixes #10197.

## Limitations

As `orig_extra.grf` is both generated and included in the repository, somebody please verify that my grfcodec produced a stable and reproducible GRF file.

## Checklist for review

Some things are not automated, and forgotten often. This list is a reminder for the reviewers.
* The bug fix is important enough to be backported? (label: 'backport requested')
* This PR touches english.txt or translations? Check the [guidelines](https://github.com/OpenTTD/OpenTTD/blob/master/docs/eints.md)
* This PR affects the save game format? (label 'savegame upgrade')
* This PR affects the GS/AI API? (label 'needs review: Script API')
    * ai_changelog.hpp, gs_changelog.hpp need updating.
    * The compatibility wrappers (compat_*.nut) need updating.
* This PR affects the NewGRF API? (label 'needs review: NewGRF')
    * newgrf_debug_data.h may need updating.
    * [PR must be added to API tracker](https://wiki.openttd.org/en/Development/NewGRF/Specification%20Status)

### CODE REVIEW

File: media/baseset/orig_extra/fix_gui_icons.nfo [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// This file is part of OpenTTD.
+// OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+// OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+// See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+//
+   -1 * 0	 0C "Fix alignment of button icons."
+
+// Fix alignment of button icons.
+   -1 * 11	 0A 03 01 DC 02 01 E2 02 01 E6 02
+   -1 sprites/fix_gui_icons.png 8bpp   8  13  18  20  1  0 normal nocrop
```
Comment (rubidium42) [inline media/baseset/orig_extra/fix_gui_icons.nfo]: Isn't the `1` meaning that it should offset the sprite? Though that information is later going to be ignored when centering, right? So shouldn't it be just 0 like the next two sprites?
And shouldn't the sprite just be made 20x20 pixels? Although I am assuming that all the other buttons are also 20x20 pixels.

Comment (michicc) [reply media/baseset/orig_extra/fix_gui_icons.nfo]: I was trying to visually match the old alignment, and just using 20 still had a pixel difference (from whatever reason).

Comment (PeterN) [reply media/baseset/orig_extra/fix_gui_icons.nfo]: Ideally it should be 20x20 with 0x0 offset. It is likely to not exactly match what it used to be before as it was incorrectly aligned then too. And the button itself is bigger due to padding being scaled.

**Comment by michicc** [comment]
State with this PR:
image

and

image

**Review by rubidium42 — APPROVED** [review]

**Pull request merged**